corehq/apps/reports/tasks.py

```diff
@@ -67,9 +67,12 @@
 
 def send_delayed_report(report_id):
     """
-    Sends a scheduled report, via  celery background task.
+    Sends a scheduled report, via celery background task.
     """
-    send_report.delay(report_id)
+    if settings.SERVER_ENVIRONMENT == 'production' and ReportNotification.get(report_id).domain == 'ews-ghana':
+        send_report_throttled.delay(report_id)
```

Comment:
Why not have a flag that can be enabled on any domain, instead of hardcoding a specific domain here? If that's not preferable, might be worth adding a note on why only this particular domain is on throttle queue

Reply:
I was hoping that this wouldn't become a widely used feature, and that we'd be able to kill it after ews-ghana gets sorted out.  I'll add a comment